### ISSUE TITLE
File size exceeds NaN byte

### ISSUE THREAD

**Issue by JoeryVandenBosch** [mcguffin/the-paste#60]
The paste  function seems to be broken for me after the update to 2.1.0. the pasted image disappears in a slit second and i get the error message File size exceeds NaN byte.
Any tips on how to resolve this?
Thank you

**Comment by JoeryVandenBosch** [comment]
Hi, i've tried in Chrome v119, in Egde v119, i'm copying from snagit of the snipping tool and it's in the classic editor. the screenshot files also don't show in the media library anymore.

**Comment by mcguffin** [comment]
> the screenshot files also don't show in the media library anymore.
You talking about previously pasted screenshots are gone from the media library, right?
Does drag-drop-upload work?

**Comment by JoeryVandenBosch** [comment]
Before when i took a screenshot and pasted it in the blog post, the screenshot file was also uploaded to the media library, this is no longer the case. All previous screenshots are still in the library. Drag and drop still works.

**Comment by mcguffin** [comment]
Okay, I'll need to examine what's on your clipboard.

Please:
1. Install the attached plugin – this version has a debug mode, which is not (yet) available in the wporg version: [the-paste.zip](https://github.com/mcguffin/the-paste/files/13473124/the-paste.zip)
2. Execute `thepaste.options.editor.debugMode=true` in the JS-console
3. Paste
4. Filter the console output by `[the paste]` and post a screenshot. Should more or less look like this:
<img width="754" alt="grafik" src="https://github.com/mcguffin/the-paste/assets/402988/862fc1a1-63d4-499b-a225-4efc5f213fe0">

**Comment by mcguffin** [comment]
68 errors in your console.
Any of this related to the plugin?

**Comment by JoeryVandenBosch** [comment]
No nothing related to the plugin

**Comment by JoeryVandenBosch** [comment]
Hi, i think i've found the reason why the plugin stopped working, In Jetpack, when you enable Videopress the plugin stopped working. If i disable Videopress all is fine again. this for the version 2.0.9 & 2.1.0 of the plugin.

**Comment by mcguffin** [comment]
Weird...
I just set up a fresh blog with jetpack, the videopress plugin (Free Trial – I'm definetly not going to pay for the work I do here!), the paste, classic editor and 2020 theme. 
No success reproduing the issue so far... 
Did you try to disable some other plugins too? Switch the Theme? Maybe it's like `videopress + something else = ding dong the paste is dead`...

**Comment by JoeryVandenBosch** [comment]
Yes I tried to disable all plugins that where added after I used the paste. Now I use Divi but i can also confirm that it did not worked with Elementor. That was some time ago and i cannot recall that Videopress was enabled at that time. But I tried the paste plugin also at that time.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
